### ISSUE TITLE
Make content_path for encrypted credentials configurable

### ISSUE THREAD

**Issue by jtrost** [rails/rails#34535]
I'm using Capistrano to deploy my Rails applications. After upgrading to Rails 5.2, I started symlinking config/credentials.yml.enc to the shared path so that it's not overwritten on each deploy. However, if I run bin/rails credentials:edit after deploying, the symlink in current/ is overwritten with the encrypted credentials file.

After reading through [credentials_command.rb](https://github.com/rails/rails/blob/3cac5fe94f0f81b4263cfa03d4822c05a55eb49c/railties/lib/rails/commands/credentials/credentials_command.rb), it seems like making `content_path` configurable would solve this problem.

### Steps to reproduce
1. Configure Capistrano to symlink current/config/credentials.yml.enc to shared/config/credentials.yml.enc.
2. Deploy the application and ensure the symlink exists.
3. From /current, run: bin/rails credentials:edit and save your changes.
4. current/config/credentials.yml.enc will now contain the actual file instead of the symlink.

### Expected behavior
We should be able to configure encrypted credentials to write to shared/config/credentials.yml.enc, and leave current/config/credentials.yml.enc alone.

### Actual behavior
The current/config/credentials.yml.enc symlink is overwritten.

### System configuration
**Rails version**: 5.2.0

**Ruby version**: 2.5.0

cc @kaspth, as I've seen you involved in most issues concerning encrypted credentials.

**Comment by kaspth** [comment]
I’m pretty sure @morgoth already added that for multi env credentials. He’d be able to confirm.

**Comment by paracycle** [comment]
@rafaelfranca If the only concern is to not overwrite symlinks, I believe the actual fix could be as simple as looking at `realpath` on the pathnames passed to the `ActiveSupport:EncryptedFile` constructor. I'd be happy to take a stab at that if you think it makes sense.

**Comment by morgoth** [comment]
Yes, credentials path is configurable since https://github.com/rails/rails/pull/33521

**Comment by jtrost** [comment]
@morgoth Unless I'm mistaken, #33521 doesn't deal with symlinks. It adds environment-specific credentials.

@paracycle Yes, the only concern here is not overwriting symlinks. Writing encrypted credentials to a file's `realpath` seems like it would solve that problem.

**Comment by kaspth** [comment]
@jtrost check the changelog entry. You can assign `Rails.application.config.credentials.content_path`. https://github.com/rails/rails/pull/33521/files#diff-1846d4882fa63eb5b1271541995cbfdcR6

Though I haven't tested it in a sample app, it should sidestep the need for looking at symlinks.
